src/mod/databases/mod_pgsql/mod_pgsql.c

```diff
@@ -779,6 +779,13 @@ switch_status_t pgsql_next_result_timed(switch_pgsql_handle_t *handle, switch_pg
 		case PGRES_PIPELINE_SYNC:
 			break;
 #endif
+//#if POSTGRESQL_MAJOR_VERSION >= 14
```

Comment:
why is this commented out?

Reply:
why is `PGRES_PIPELINE_SYNC` mentioned twice?